centos-7/cloud-init/95_exoscale.cfg

```diff
@@ -0,0 +1,28 @@
+# dicts are merged with the dicts defined in the main cloud.cfg file.
+# WARNING: lists (in dict values) are NOT merged but OVERWRITTEN
+
+# Configuration modules (run once per instance)
+# NOTE: The 'set-passwords' module must be run at every boot to work along
+#       Exoscale password reset feature
+cloud_config_modules:
+ - emit_upstart
+ - snap
+ - ssh-import-id
+ - locale
+ - [set-passwords, always]
+ - grub-dpkg
+ - apt-pipelining
```

Comment:
You may want to remove the debian/ubuntu related modules like:
- snap
- apt-*
- ubuntu-*
- byobu
- etc...